### PR TITLE
api: use structured logging to improve log readability

### PR DESCRIPTION
**Pull request by henrybarreto**

### CODE REVIEW

File: api/store/mongo/migrations/migration_10.go [hunk 1]
```diff
@@ -4,6 +4,7 @@ import (
 	"context"
 
 	"github.com/sirupsen/logrus"
+
```
Comment (otavio) [inline api/store/mongo/migrations/migration_10.go:7]: This seems to be an unrelated change. Please drop it.

**Review by otavio — CHANGES_REQUESTED** [review]
@henrybarreto it seems there are some import-related changes. Are those intentional or due to linter?

File: api/store/mongo/migrations/migration_10.go (hunk 1 above)
Comment (henrybarreto) [reply api/store/mongo/migrations/migration_10.go:7]: This behavior is due to IntelliJ auto import. I did not note that.
Fixing it now

Comment (otavio) [reply api/store/mongo/migrations/migration_10.go:7]: We try to keep the changes at a minimum, thanks for reviewing it.

**Review by otavio — APPROVED** [review]

File: api/store/mongo/migrations/migration_1.go [hunk 2]
```diff
@@ -10,12 +10,20 @@ var migration1 = migrate.Migration{
 	Version:     1,
 	Description: "Create the database for the system",
 	Up: func(db *mongo.Database) error {
-		logrus.Info("Applying migration 1 - Up")
+		logrus.WithFields(logrus.Fields{
+			"component": "migration",
+			"version":   1,
+			"action":    "Up",
```
Comment (otavio) [inline api/store/mongo/migrations/migration_1.go:16]: Looking at the task description @gustavosbarreto did, he used lowercase for _up_ and _down_. This seems better indeed.

**Review by otavio — CHANGES_REQUESTED** [review]
